registers/bin/generator/src/main.rs

```diff
@@ -41,6 +41,8 @@ static CALIPTRA_RDL_FILES: &[&str] = &[
     "src/integration/rtl/caliptra_reg.rdl",
 ];
 
+static I3C_CORE_RDL_FILES: &[&str] = &["src/rdl/registers.rdl"];
```

Comment:
I was under the impression that Caliptra Core would just see the recovery interface and not bus-specific details. Could you point me at any spec for what this i3c RTL in Caliptra Core is for?

Reply:
So the recovery interface is part of the i3c periph. The intention is indeed to only use that bus abstracted part.
SecFwRecoveryIf in https://github.com/chipsalliance/i3c-core/tree/main/src/rdl is that interface we'll be targetting.

I think the other capabilities like TTI (I3C Target) will be used by MCU.

Reply:
Ok, thanks for the details. @bharatpillilli would it make sense to factor the recovery interface RTL to be independent from the i3c peripheral RTL?

Not a huge deal, just thinking it makes the organization more clear for what Caliptra Core depends on.